### PR TITLE
Create current-drug-targets.csl

### PR DESCRIPTION
**Pull request by AKREMA007**
The changes have been made according to current drug targets journal of bentham science.

### CODE REVIEW

File: current-drug-targets.csl [hunk 1]
```diff
@@ -0,0 +1,201 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" page-range-format="minimal" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>Current Drug Targets</title>
+    <title-short>Curr Drug Tar</title-short>
+    <id>http://www.zotero.org/styles/current-drug-targets</id>
+    <link href="http://www.zotero.org/styles/current-drug-targets" rel="self"/>
+    <link href="http://www.zotero.org/styles/elsevier-vancouver" rel="template"/>
+    <link href="http://www.elsevier.com/journals/journal-of-crohns-and-colitis-supplements/1873-9954/guide-for-authors" rel="documentation"/>
```
Comment (rmzelle) [inline current-drug-targets.csl]: @AKREMA007, could you update the "template" link (should point to "journal-of-crohns-and-colitis-supplements"), "documentation" link ("http://benthamscience.com/journals/current-drug-targets/author-guidelines/"), ISSNs (1873-5592 (Online), 1389-4501 (Print)), and summary?

File: current-drug-targets.csl [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+      <group delimiter=": ">
+        <text variable="publisher-place"/>
+        <text variable="publisher"/>
+      </group>
+      <text macro="year-date"/>
+    </group>
+  </macro>
+  <macro name="publisher_thesis">
+    <group delimiter=", ">
+      <text variable="publisher" form="long"/>
```
Comment (adam3smith) [inline current-drug-targets.csl]: remove the `form="long"` here. It has no function

File: current-drug-targets.csl [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+  </citation>
+  <bibliography entry-spacing="0" second-field-align="flush" et-al-min="11" et-al-use-first="10">
+    <layout suffix=".">
+      <text variable="citation-number" prefix="[" suffix="]"/>
+      <text macro="author" suffix=" "/>
+      <choose>
+        <if type="bill book graphic legal_case legislation motion_picture report song" match="any">
+          <group delimiter=". ">
+            <text variable="title" form="long" font-style="italic"/>
+            <text variable="volume" prefix="vol. " form="long"/>
```
Comment (adam3smith) [inline current-drug-targets.csl]: remove the `form="long"` here, too.

File: current-drug-targets.csl [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+            <group delimiter=" ">
+              <label variable="page" form="short" plural="never"/>
+              <text variable="page" form="short"/>
+            </group>
+          </group>
+        </if>
+        <else-if type="chapter paper-conference" match="any">
+          <group delimiter=". ">
+            <group delimiter=". ">
+              <text variable="title" form="long"/>
```
Comment (adam3smith) [inline current-drug-targets.csl]: here, too

File: current-drug-targets.csl [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+            <text macro="publisher"/>
+            <group delimiter=" ">
+              <label variable="page" form="short" plural="never"/>
+              <text variable="page" form="short"/>
+            </group>
+          </group>
+        </if>
+        <else-if type="chapter paper-conference" match="any">
+          <group delimiter=". ">
+            <group delimiter=". ">
```
Comment (adam3smith) [inline current-drug-targets.csl]: you can remove one of these groups and then remove the group end tag in line 150. They're redundant.

**Comment by rmzelle** [comment]
@AKREMA007, let us know if you need any help making these changes.

**Comment by rmzelle** [comment]
Closing because of inactivity. @AKREMA007, you can still make the requested changes; if you do that and leave a comment, we'll take another look.

**Pull request closed without merging**